pkg/messages/messages_en.go

```diff
@@ -143,6 +143,12 @@ var messagesEn = map[string]Message{
 		Type:        codes.Internal,
 		Action:      "Please check if there is any error in POD describe related with volume attach",
 	},
+	DevicePathNotFound: {
+		Code:        DevicePathNotFound,
+		Description: "Device path '%s' is not present",
+		Type:        codes.Internal,
+		Action:      "List volume attachments by using `ibmcloud ks storage attachments --worker <worker-ID> --cluster <cluster-ID> | grep <volume-ID≥` If the volume is attached, open a ticket and select VPC for the Problem type. Otherwise, select IBM Cloud Kubernetes service Problem type.",
```

Comment:
Just a small observation, `grep <volume-ID≥` should be `grep <volume-ID>` and end with a full stop. Something like

```
"List volume attachments by using `ibmcloud ks storage attachments --worker <worker-ID> --cluster <cluster-ID> | grep <volume-ID>`. If the volume is attached, open a ticket and select VPC for the Problem type. Otherwise, select IBM Cloud Kubernetes service Problem type."
```